python/observation-publisher/publisher/observation_sink.py

```diff
@@ -275,10 +278,10 @@ def schema_fields(self) -> List[SchemaField]:
 
     @property
     def write_location(self) -> str:
```

Comment:
nit: I think it would be nice to have Python docstring here explaining the generated write_location. At first, it might be confusing when user try to differentiate between CaraML project and BQ project.